src/models/component.ts

```diff
@@ -24,6 +24,7 @@ export type Component<T extends ComponentType> = {
   name: string
   stack: StackFor<T>
   type: T
+  others?: any
```

Comment:
I don't like this, it's too vague. Can't we just map only the props we need explicitly? mfeType, configMfe if I got it well